nextflow-parsing/NextflowParsingFunction/src/test/java/io/dockstore/nextflowparsing/AppTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dockstore.nextflowparsing;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dockstore.openapi.client.model.LanguageParsingRequest;
+import io.dockstore.openapi.client.model.LanguageParsingResponse;
+import java.net.HttpURLConnection;
+import javax.ws.rs.core.MediaType;
+import org.junit.jupiter.api.Test;
+
+public class AppTest {
+
+  @Test
+  public void successfulResponse() throws Exception {
+    LanguageParsingRequest request = new LanguageParsingRequest();
+    request.setBranch("2.3");
+    request.setUri("https://github.com/SciLifeLab/Sarek.git");
```

Comment:
code comment that this is an archived repo and thus we don't have to worry about changes on line 58 and below

Reply:
random thought, since this isn't "our" archived repo, should we fork and archive it in case they delete the repo